extensions/ql-vscode/src/common/short-paths.ts

```diff
@@ -0,0 +1,111 @@
+import { platform } from "os";
```

Comment:
Which repo should we track the tech debt in?
Any unit test we write would require manual machine configuration to run on a typical developer machine, since 8.3 name generation has been off by default for the last decade or so. It would still run in CI, though, since the Actions runners seem to have that feature enabled.

Reply:
I'll raise an issue to cover this